binaries/cuprated/src/blockchain/fast_sync.rs

```diff
@@ -0,0 +1,24 @@
+use std::slice;
+
+use cuprate_helper::network::Network;
+
+/// The hashes of the compiled in fast sync file.
+static FAST_SYNC_HASHES: &[[u8; 32]] = unsafe {
+    let bytes = include_bytes!("./fast_sync/fast_sync_hashes.bin");
+
+    #[expect(clippy::manual_assert, reason = "assert is not const")]
+    if bytes.len() % 32 != 0 {
+        panic!()
+    }
+
+    slice::from_raw_parts(bytes.as_ptr().cast::<[u8; 32]>(), bytes.len() / 32)
+};
```

Comment:
`bytemuck::cast_slice` almost certainly works here although unfortunately the const `bytemuck::must_cast_slice` seems to trip up.

```suggestion
/// The hashes of the compiled in fast sync file.
static FAST_SYNC_HASHES: &[[u8; 32]] = {
    let bytes = include_bytes!("./fast_sync/fast_sync_hashes.bin");

    if bytes.len() % 32 == 0 {
        // SAFETY: The file byte length must be perfectly divisible by 32, checked above.
        unsafe { slice::from_raw_parts(bytes.as_ptr().cast::<[u8; 32]>(), bytes.len() / 32) }
    } else {
        panic!();
    }
};
```